internal/nodes/dequeue_ingestion.go

```diff
@@ -263,9 +263,11 @@ func DequeueIngestion(key string, fullsync bool) {
 				} else {
 					aviModelGraph := NewAviObjectGraph()
 					aviModelGraph.BuildAdvancedL4Graph(namespace, gwName, key, false)
-					ok := saveAviModel(modelName, aviModelGraph, key)
-					if ok && len(aviModelGraph.GetOrderedNodes()) != 0 && !fullsync {
-						PublishKeyToRestLayer(modelName, key, sharedQueue)
+					if len(aviModelGraph.GetOrderedNodes()) > 0 {
+						ok := saveAviModel(modelName, aviModelGraph, key)
+						if ok && len(aviModelGraph.GetOrderedNodes()) != 0 && !fullsync {
```

Comment:
`len(aviModelGraph.GetOrderedNodes()) != 0` is already taken care in L266 it seems.